lib/crowbar/init/application.rb

```diff
@@ -392,9 +392,7 @@ class Application < Sinatra::Base
 
           if init[:body] # nil body means success
             logger.error(init[:body])
-            res[:errors] = {
-              data: init[:body]
-            }
+            res[:errors] = init[:body]
```

Comment:
crowbar_init return is already formatted as expected. An alternative would be to do:

```
res[:errors] = {
  data: init[:body][:data]
}
```
Maybe this will be more clear.